fake testers_file_path

```diff
@@ -0,0 +1 @@
+First,Last,Email,Groups,Installed Version,Install Date
```

Comment:
Was this file uploaded by mistake? Do we need it?

Reply:
fixed! it is autogenerated file by unit tests. i will be it properly in separate PR https://github.com/fastlane/fastlane/pull/20631/commits/13afc8ccf98878de690b5568ea33de7a3c226dae

Reply:
Shouldn't the unit tests generate such files in a temporary dir instead on generating them at the repo root? 🤔 

Though I think that file is not generated by any test that you added in this PR, but has instead always been generated there, by another test that was already there even before your PR, right? In that case, fixing that is outside the scope of this PR; but maybe something to keep in mind for a later fix? 🤷